src/test/java/io/xstefank/wildfly/bot/PRRuleDirectoryHitTest.java

```diff
@@ -50,19 +50,19 @@ void testDirectoriesMentionsNewFileInDiff() throws IOException {
             """;
 
         given().github(mocks -> {
-                    mocks.configFile(CONFIG_FILE_NAME).fromString(wildflyConfigFile);
-                    MockedGHPullRequestProcessor.processPullRequestMock(mocks.pullRequest(gitHubJson.id()),
-                            mockFileDetails(), MockedGHPullRequestProcessor.mockEmptyComments());
-                })
-                .when().payloadFromString(gitHubJson.jsonString())
-                .event(GHEvent.PULL_REQUEST)
-                .then().github(mocks -> {
-                    GHPullRequest mockedPR = mocks.pullRequest(gitHubJson.id());
-                    Mockito.verify(mockedPR, Mockito.times(2)).listFiles();
-                    Mockito.verify(mockedPR).comment("/cc @7125767235");
-                    Mockito.verify(mockedPR, Mockito.times(1)).listComments();
-                    Mockito.verifyNoMoreInteractions(mockedPR);
-                });
+                mocks.configFile(CONFIG_FILE_NAME).fromString(wildflyConfigFile);
```

Comment:
https://github.com/xstefank/wildfly-github-bot/pull/112#discussion_r1290987969

Reply:
I know :) working on it